notebooks/tutorial_ivf_pq.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    import matplotlib.pyplot as plt
</pre><p>Please put all the includes together in this cell. This tells a user right away whether or not the notebook is going to work, rather than them getting to the 5th cell and realizing they don't have something installed, just to fail at 6th cell when they realize they didn't have something else. We are going to provide a separate conda environment yaml file for them to set up their environment for running notebooks as well, but having all the imports up front will immediately tell them whether the notebook is going to work. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>